core/src/main/java/com/datastax/oss/driver/api/core/config/DefaultDriverOption.java

```diff
@@ -689,6 +689,34 @@ public enum DefaultDriverOption implements DriverOption {
    * <p>Value-type: boolean
    */
   PREPARE_ON_ALL_NODES("advanced.prepared-statements.prepare-on-all-nodes"),
+
+  /**
+   * CQL 4.x has a known issue where prepared statement invalidation may be bypassed on the client
+   * side. Reference: https://github.com/scylladb/scylladb/issues/20860
+   *
+   * <p>When this occurs, the client's metadata can become outdated, leading to various
+   * deserialization errors.
+   *
+   * <p>To mitigate this, the driver can disable the `skip metadata` flag, ensuring the server
+   * includes metadata with every query response.
+   *
+   * <p>This setting determines how the driver handles the `skip metadata` flag for CQL 4 prepared
+   * statements: - **"smart"** (default) – Disables the flag only for wildcard selects (`SELECT *
+   * FROM`) and queries that return UDTs (including UDT collections and maps containing UDTs). -
+   * **"always-on"** – Enables the `skip metadata` flag, preventing metadata from being sent. -
+   * **"always-off"** – Disables the `skip metadata` flag, ensuring metadata is included in every
```

Comment:
Since the options are limited there it would be nice to have them represented as an enum, similarly to other config options. Consistency levels should be a similar case to look for reference.